src/Service/ApplicationService.cs

```diff
@@ -0,0 +1,24 @@
+using Passwordless.Service.Helpers;
+using Passwordless.Service.Models;
+using Passwordless.Service.Storage.Ef;
+
+namespace Passwordless.Service;
+
+public class ApplicationService : IApplicationService
+{
+    private readonly ITenantStorage _storage;
+    public ApplicationService(ITenantStorage storage)
+    {
+        _storage = storage;
+    }
+
+    public async Task SetFeaturesAsync(SetFeaturesDto features)
+    {
+        if (features == null)
+        {
+            throw new ApiException("No 'body' or 'parameters' were passed.", 400);
+        }
+
+        await _storage.SetFeaturesAsync(features);

```

Comment:
Since this is callable with SecretKey, we would need to add some constraints on the `SetFeaturesDto` right? I suggest adding attributes Min/max and make use of the MiniValidation that Jared recently added.